### PR TITLE
completed Task 1

### PR DESCRIPTION
**Pull request by kooldandy**

### CODE REVIEW

File: Task1.1/task1.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const through2 = require("through2");
```
Comment (rchawla17) [inline Task1.1/task1.js:1]: Please use ES6 and try using imports.

File: Task1.1/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nodejsglobalmentoringprogram",
+  "version": "1.0.0",
+  "description": "Node Js Mentoring",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (rchawla17) [inline Task1.1/package.json:7]: Add script for babel for compiling your ES6 code.

File: Task1.1/task1.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const reverseString = () => {
+  console.log("Please enter any string to reverse.\n");
+  process.stdin
+    .pipe(
+      through2(function write(buffer, encoding, next) {
```
Comment (rchawla17) [inline Task1.1/task1.js:7]: Can you do this without the use of external modules?

File: Task1.2/modules/directoryWatcher.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const EventEmitter = require("events");
+const fs = require("fs");
+const csvtojson = require("csvtojson");
+
+class DirectoryWatcher extends EventEmitter {
+  
+  checkDirectoryForFiles (sourceFilePath, destinationFilePath) {
+    const file = fs.readdirSync(sourceFilePath)[0];
```
Comment (rchawla17) [inline Task1.2/modules/directoryWatcher.js:8]: If you're adding index, why do you need to create a directory? Why don't you directly refer the name of the file. If you're intending on creating the directory, you should consider making this dynamic.

File: Task1.2/modules/index.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const DirectoryWatcher = require("./directoryWatcher");
```
Comment (rchawla17) [inline Task1.2/modules/index.js:1]: use ES6

File: Task1.3/package.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-stage-2": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "nodemon": "^1.19.1"
+  },
+  "dependencies": {
+    "csvtojson": "^2.0.10",
+    "request": "^2.88.2",
```
Comment (rchawla17) [inline Task1.3/package.json]: Why did you use `request` ?

File: Task1.1/task1.js (hunk 1 above)
Comment (kooldandy) [reply Task1.1/task1.js:1]: Task 1 was in commeonJS module so didn't use it

File: Task1.2/modules/index.js (hunk 5 above)
Comment (kooldandy) [reply Task1.2/modules/index.js:1]: Task 2 was in commonJs module

File: Task1.1/package.json (hunk 2 above)
Comment (kooldandy) [reply Task1.1/package.json:7]: Not required

File: Task1.3/package.json (hunk 6 above)
Comment (kooldandy) [reply Task1.3/package.json]: Removed

File: Task1.1/task1.js (hunk 3 above)
Comment (kooldandy) [reply Task1.1/task1.js:7]: Fixed

**Review by rchawla17 — APPROVED** [review]

**Pull request merged**